pkg/binding/binding.go

```diff
@@ -109,7 +109,8 @@ func (c *Controller) syncBinding(ctx context.Context, bindingName string) error
 	policyWithStatus := policy.DeepCopy()
 	policyWithStatus.Status = v1alpha1.BindingPolicyStatus{
 		ObservedGeneration: policy.Generation,
-		Errors:             policyErrors,
+		Conditions:         binding.Status.Conditions,
+		Errors:             append(policyErrors, binding.Status.Errors...),
```

Comment:
`binding.Status.Errors` were already added at line 91.
I think that it would be fine to change line 91 to initialize `policyErrors` to an empty slice.

Reply:
Done.